catalog/app/components/Preview/Preview.js

```diff
@@ -15,10 +15,12 @@ const fallback = {
 };
 
 const loaderChain = [
+  loaders.Csv,
```

Comment:
fine for now, but i question the casing. we have csv, CSV, Csv, etc. all in the same combined codebase

Reply:
we use the js CamelCase (not sure if it's 100% precise term) convention here, abbreviations are considered words, so only the first letter should be capitalized, e.g. `JsonEncoder`. it's not 100% consistent across the codebase, but i'm trying to adhere to that convention.